app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -99,17 +105,16 @@ public static void open(@NonNull final Context context) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(layout.activity_search);
+        ButterKnife.bind(this);
         Option.ofObj(savedInstanceState)
               .ifNone(this::addSearchFragment);
 
-        Toolbar toolbar = (Toolbar) findViewById(id.toolbar_search);
+        Toolbar toolbar = ButterKnife.findById(this, id.toolbar_search);
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
-        searchView = get((SearchView) findViewById(id.search_view));
-        searchView.setIconified(false);
-        closeButton = (ImageView) searchView
-                .findViewById(android.support.v7.appcompat.R.id.search_close_btn);
+        get(searchView).setIconified(false);
```

Comment:
I wonder if we could start applying the same logic to Dagger injections too, as I understand it they can't be injected as null either - isn't there now some generated code that prohibits null unless otherwise specified?